pkg/bench/rttanalysis/orm_queries_bench_test.go

```diff
@@ -285,6 +285,66 @@ ORDER BY i.relname`,
 FROM indexes
 ORDER BY relname DESC, input`,
 		},
+
+		{
+			Name:  "hasura column descriptions",
+			Setup: "CREATE TABLE t(a INT PRIMARY KEY)",
+			Stmt: `WITH
+  "tabletable" as ( SELECT "table".oid,
+           "table".relkind,
+           "table".relname AS "table_name",
+           "schema".nspname AS "table_schema"
+      FROM pg_catalog.pg_class "table"
+      JOIN pg_catalog.pg_namespace "schema"
+          ON schema.oid = "table".relnamespace
+      WHERE "table".relkind IN ('r', 't', 'v', 'm', 'f', 'p')
+        AND "schema".nspname NOT LIKE 'pg_%'
+        AND "schema".nspname NOT IN ('information_schema', 'hdb_catalog', 'hdb_lib', '_timescaledb_internal', 'crdb_internal')
+  )
+SELECT
+  "table".table_schema,
+  "table".table_name,
+  coalesce(columns.description, '[]') as columns
+FROM "tabletable" "table"
+
+LEFT JOIN LATERAL
+  ( SELECT
+      pg_catalog.col_description("table".oid, "column".attnum) as description
```

Comment:
I think the fix here is to vectorize this builtin so we can do the fetching in parallel. At least, I think that's the simplest approach. @chengxiong-ruan is working on cleaning up some of the comment access but I don't think it's going to be easy to leverage that to pre-fetch the right comments.

Reply:
I think it'd be nice to have some more tests with more tables and more columns so that the growth in terms of round-trips is clear.

Reply:
I don't think there is any growth in round-trips based on number of tables

Reply:
Can you prove me wrong? I expect for every column in every table that we'll invoke `pg_catalog.col_description` which will query system.comments, no?

Reply:
oh to be clear, i made that comment after modifying the benchmark setup and trying it, and observing no change. i can commit that though if it's worthwhile

Reply:
also to be clear, i don't really understand why it didn't change

Reply:
Hmm I wonder if we have a bug in rttanalysis!

Reply:
I feel like the apply join trace doesn't propagate correctly.

Reply:
I tried it out and it seems like what's going on is that the trace is so big that it's being truncated 😓

Reply:
ah darn... well then i will go ahead and merge this for now, and if are able to workaround that truncation in the future we can update the benchmark at that time.

Reply:
https://github.com/cockroachdb/cockroach/issues/90961